CONTRIBUTING.md

```diff
@@ -1,14 +1,27 @@
+Issue report guidelines
```

Comment:
I think it will make issue reporting clunky. Every issue is different, templates make too much hassle to read, fill in, if not matching to my issue, remove parts of, etc.